### ISSUE TITLE
How to fine-tune your pretrained model

### ISSUE THREAD

**Issue by taojian1989** [antoine77340/Youtube-8M-WILLOW#17]
Hi,could you tell me how to modify code if I want to fine-tune the pretrained model you provided （gatednetvladLF-256k-1024-80-0002-300iter-norelu-basic-gatedmoe）,my own dataset has only 12 classes.
Thanks.

**Comment by orprager** [comment]
you can transform all train tfrecords labels to your labels and then re-train willow

**Comment by Abelonx** [comment]
> you can transform all train tfrecords labels to your labels and then re-train willow

but usually, our dataset is much less than youtube8m, re-train willow seems impossible

**Comment by ShiqiLi047** [comment]
> > you can transform all train tfrecords labels to your labels and then re-train willow
> 
> but usually, our dataset is much less than youtube8m, re-train willow seems impossible

Did you re-train willow? I am trying re-train willow. However, it seems does not work.
